pkgs/tools/networking/oauth2ms/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchFromGitHub, python3 }:
+
+stdenv.mkDerivation rec {
```

Comment:
Can we use the `finalAttrs` pattern?

Reply:
Converted to `finalAttrs`. Is there any advantage to this if the derivation does not refer to any of its parameters?

Reply:
@wentasah No. In fact it may be ideal to simply remove the `rec` part instead of converting it to a `finalAttrs` function. I would suggest doing that and testing if it still works as intended. If so, keeping it simple is probably better.

Reply:
The advantage over simply removing `rec` would be a smaller diff if, in the future, there was a need to refer to an attribute, which is not terribly significant imo. No need to remove it if it's already present though.

Reply:
I have removed it. Having smaller diff in the future is just a theoretical advantage. Moreover, I guess that `finalAttrs` increases eval time a bit. Overall eval time is already too long so it's better not to increase it.